kore/src/Kore/Step/Simplification/Ceil.hs

```diff
@@ -361,3 +362,66 @@ makeEvaluateNormalizedAc
         }
   | Map.null concreteElements = Just $ makeEvaluateTerm sideCondition opaqueAc
 makeEvaluateNormalizedAc _  _ = Nothing
+
+makeSimplifiedCeil
```

Comment:
Could you add a comment here explaining that this is the terminal case of the `Ceil` simplifier, and that it adds the `Ceil` of subterms, as required?